internal/functions/serve/serve.go

```diff
@@ -217,6 +217,7 @@ func populatePerFunctionConfigs(cwd, importMapPath string, noVerifyJWT *bool, fs
 	if err != nil {
 		return nil, "", err
 	}
+	functionsConfig = deploy.FilterFunctionsToDeploy(functionsConfig)
```

Comment:
Here I wonder, should the flag also skip the serve of those functions locally ?

Reply:
Yes, I would think so. We want to have functions served locally to match those deployed.